heartbeat/aws-vpc-move-ip

```diff
@@ -213,25 +224,35 @@ ec2ip_validate() {
 }
 
 ec2ip_monitor() {
-        MON_RES=""
+	MON_RES=""
+	if ocf_is_true ${OCF_RESKEY_eni_id_lookup}; then
+		EC2_ATTRIBUTE_LOOKUP="NetworkInterfaceId"
+    	EC2_ID="$(ec2ip_get_instance_eni)"
+    	identification="interface"
+  	else
+    	EC2_ATTRIBUTE_LOOKUP="InstanceId"
+    	EC2_ID="$EC2_INSTANCE_ID"
+    	identification="instance"
```

Comment:
Any reason for mixing cases in the variable names (all caps for the `EC2_*` ones and all lowercase for `identification`)?

Reply:
And this is only a pretty minor point on my part: To me, "identification" sounds like something unique to the resource (instance/interface) that we're looking up. This variable seems more like a "resource_type", or something similar.

Reply:
updated